xrpl/models/transactions/vault_deposit.py

```diff
@@ -0,0 +1,26 @@
+"""Represents a VaultDeposit transaction on the XRP Ledger."""
+
+from dataclasses import dataclass, field
+
+from xrpl.models.amounts import Amount
+from xrpl.models.required import REQUIRED
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.models.transactions.types import TransactionType
+from xrpl.models.utils import KW_ONLY_DATACLASS, require_kwargs_on_init
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True, **KW_ONLY_DATACLASS)
+class VaultDeposit(Transaction):
```

Comment:
Should also check for valid vault_id here

Reply:
fixed in [3661d9a](https://github.com/XRPLF/xrpl-py/pull/814/commits/3661d9ac4477a42dae041eec10bb6b1c0c76735d)